File: Managed/UnrealSharp/UnrealSharp/Interop/FRotatorExporter.cs
using UnrealSharp.Binds;
using UnrealSharp.CoreUObject;

namespace UnrealSharp.Interop;

[NativeCallbacks]
public static unsafe partial class FRotatorExporter
{
    public static delegate* unmanaged<out FRotator, ref FMatrix, void> FromMatrix;
}

File: Source/UnrealSharpCore/Private/Export/FRotatorExporter.cpp
#include "CSBindsManager.h"

DECLARE_UNREALSHARP_EXPORTER(FRotatorExporter)
{
	void FromMatrix(FRotator* Rotator, const FMatrix& Matrix)
	{
		*Rotator = Matrix.Rotator();
	}
	
	EXPORT_UNREALSHARP_FUNCTION(FromMatrix)
}




